backend/person-service/src/main/kotlin/com/nosql/personservice/service/impl/DefaultPersonService.kt

```diff
@@ -73,4 +74,12 @@ class DefaultPersonService(
 
         return conversionService.convert(personRecord, PersonDto::class)
     }
+
+    override suspend fun edit(personId: String, personDto: PersonDto): PersonDto {
+        val personRecord = personComponent.get(ObjectId(personId))
+            .apply { merge(personDto) }
+            .let { personComponent.update(it) }
```

Comment:
I think you should use `conversionService` to convert `PersonDto` to `PersonEntity` and then use `personComponent` for updating `Person` via `merge` entity with entity.

Reply:
fixed, and in contacts